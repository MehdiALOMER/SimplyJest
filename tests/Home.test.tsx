import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../app/index';

describe('Home Screen', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Home />).toJSON();
        expect(tree).toMatchSnapshot(); // Snapshot testi
    });
});

// npx jest tests/Home.test.tsx