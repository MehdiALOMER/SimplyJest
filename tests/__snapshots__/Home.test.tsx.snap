// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen renders correctly 1`] = `
<View>
  <Text>
    Welcome to the Updated Unit Test Demo1!
  </Text>
</View>
`;
